nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -401,6 +401,73 @@ define([
             controlPanel.setExecMessage(Jobs.createCombinedJobState(jobState));
         }
 
+        /* JOB MANAGEMENT */
+
+        /**
+         * Cancel a single job from the batch
+         *
+         * @param {string} jobId
+         */
+        function cancelJob(jobId) {
+            // TODO: implement
+            alert(`Cancel job ${jobId}`);
+            // runtime.bus().emit('request-job-cancellation', {
+            //     jobId: jobId,
+            // });
+        }
+
+        /**
+         * Cancel all jobs with the specified status
+         *
+         * @param {string} status
+         */
+        function cancelJobsByStatus(status) {
+            const validStates = ['created', 'estimating', 'queued', 'running'];
+            if (!validStates.includes(status)) {
+                throw new Error(`${status} is not a valid job state`);
+            }
+            // TODO: implement
+            alert(`Cancel jobs with status ${status}`);
+        }
+
+        /**
+         * Retry a single job from the batch
+         *
+         * @param {string} jobId
+         */
+        function retryJob(jobId) {
+            // TODO: implement
+            alert(`Retry job ${jobId}`);
+        }
+
+        /**
+         * Retry all jobs that ended with the specified status
+         *
+         * @param {string} status
+         */
+        function retryJobsByStatus(status) {
+            const validStates = ['error', 'terminated'];
+            if (!validStates.includes(status)) {
+                throw new Error(`${status} is not a valid job state`);
+            }
+            // TODO: implement
+            alert(`Retry jobs with status ${status}`);
+        }
+
+        const jobManager = {
+            cancelJob,
+            cancelJobsByStatus,
+            retryJob,
+            retryJobsByStatus,
+        };
+
+        // add in an alias to switch to the results panel for the job status table
+        jobManager.viewJobResults = () => {
+            toggleTab('results');
+        };
```

Comment:
gather all this job-related stuff into an object that can easily be passed to different tabs (and the components of those tabs)